### PR TITLE
chore: [SIW-1706] Removed PDV service

### PR DESCRIPTION
**Pull request by silvicir**
#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-wallet-user-func/src/infra/http/handlers/__test__/create-wallet-attestation.spec.ts [hunk 1]
```diff
@@ -193,7 +188,7 @@ describe("CreateWalletAttestationHandler", async () => {
             isRevoked: true,
             revokedAt: new Date(),
             signCount: 0,
-            userId: "123" as NonEmptyString,
+            userId: "AAA" as FiscalCode,
```
Comment (grausof) [inline apps/io-wallet-user-func/src/infra/http/handlers/__test__/create-wallet-attestation.spec.ts:191]:
```suggestion
            userId: "AAACCC94E17H501P" as FiscalCode,
```

Or better
`const mockFiscalCode = "AAACCC94E17H501P"`

File: apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-attestation.ts [hunk 2]
```diff
@@ -29,45 +28,44 @@ const requireWalletAttestationRequest = (req: H.HttpRequest) =>
...
-        ),
-        RTE.fromTaskEither,
+    req,
+    requireWalletAttestationRequest,
+    TE.fromEither,
+    TE.chain((payload) =>
+      pipe(
+        payload.assertion,
+        verifyWalletAttestationRequest,
+        TE.map((assertion) => ({ assertion, fiscalCode: payload.fiscalCode })),
```
Comment (grausof) [inline apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-attestation.ts:48]: `assertion` is a parameter of **WalletAttestationRequest** which is a defined term within the technical rules. I would leave `walletAttestationRequest` or `validWalletAttestationRequest`. what do you think?

File: apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts [hunk 3]
```diff
@@ -40,15 +40,11 @@ const requireWalletInstanceRequest = (req: H.HttpRequest) =>
...
-        req,
-        requireWalletInstanceRequest,
-        RTE.fromEither,
-      ),
-    }),
-    RTE.chainW(({ user, walletInstanceRequest }) =>
+    req,
+    requireWalletInstanceRequest,
+    RTE.fromEither,
+    RTE.map((x) => x),
```
Comment (grausof) [inline apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts:46]: Why?

Comment (silvicir) [reply apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts:46]: 😄

File: apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-attestation.ts (hunk 2 above)
Comment (grausof) [reply apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-attestation.ts:48]: It was just a nitpick, no problem, it's fine to leave it like this too

**Review by grausof — APPROVED** [review]

**Pull request merged**